ichnaea/content/templates/terms.pt

```diff
@@ -0,0 +1,204 @@
+            education, recreation, social, or non-commercial technology.
+        </dd>
+    </dl>
+
+    <h3>Eligibility &amp; Registration</h3>
+
+    <p class="text-justified">
+        Unless otherwise stated in these Terms, the MLS Query API is available
+        as follows:
+    </p>
+
+    <ol>
+        <li>
+            <span class="legal-term">Commercial Project licenses</span>
+            are capped at a daily usage limit of 100,000 requests (this will
+            not be increased).  API keys expire automatically after three
+            months.
+        </li>
+        <li>
+            <span class="legal-term">Public Interest Project licenses</span>
+            are capped at a daily usage limit of 100,000 requests, which may be
+            increased on request if necessary for the project.
+        </li>
+    </ol>
+
+    <p class="text-justified">
+        Registration is required to access the MLS Query API.  Whether a
+        project meets the eligibility criteria for a license is at Mozilla’s
+        sole discretion.
+    </p>
+
+    <h3>Your Obligations</h3>
+
+    <p class="text-justified">
+        You represent and warrant that:
+    </p>
+
+    <ol>
+        <li>
+            <span class="legal-term">Relationship with Mozilla</span>.
+            You will provide accurate information and updates to Mozilla.
+            Unless you and Mozilla have a written agreement stating otherwise,
+            your project will not imply in any way that it is produced,
+            sponsored or endorsed by Mozilla.
+        </li>
+        <li>
+            <span class="legal-term">Relationship with End-Users</span>.
+            You are solely responsible for your project and obtaining
+            meaningful consent of your End-Users to locate them. You will
+            provide your End-Users with notice of your use of the MLS Query
+            API, as well as any other relevant location data you have or
+            collect in connection with its use.
+        </li>
+    </ol>
+
+    <h3>Privacy Notice &amp; Communications</h3>
+
+    <p class="text-justified">
+        As part of using the MLS Query API, you agree to the following:
+    </p>
+
+    <ol>
+        <li>
+            <span class="legal-term">Data Collection</span> -
+            We adhere to the practices outlined in the
+            <a href="https://www.mozilla.org/privacy/">Mozilla privacy policy</a>
+            for how we receive, handle and share information we collect
+            providing MLS. The following data is required for use of our
+            service: Name, Email, Organization (as part of registration). To
+            determine location, end user devices send us a combination of IP
+            address and nearby public Wi-fi networks and cell phone towers,
+            which we use to provide an approximate location. We do not keep
+            this data or track users. Mozilla retains aggregate data of the
```

Comment:
This isn’t quite right. All user queries are stored and used to revalidate the available data. This data isn’t associated with any form of user identifier, but the data itself is stored and used.

Reply:
The queries we're talking about here.... are they simply used to check the DB to see if we have that point already, bumping last_seen if so and adding it if not?

Reply:
The last part of https://mozilla.github.io/ichnaea/install/architecture.html#web-service should be correct. The data is used to validate and change the position estimate of existing data. It also adds new unknown networks found in the query. In order to do this, all queries get stored in Redis for a bit. Whether or not those also get dumped into an S3 bucket for long term storage ... I don't remember.

That long term storage decision is a policy encoded in the ``export_config`` database table and not in code. In it are a set of export targets, like s3 and potential commercial partners. That table also has a set of exclusions for both API keys and sources of data. But I don't remember what those exclusions looked like anymore.

Reply:
Thanks @hannosch! This looks like the relevant bit from the architecture docs:

> Should the service be able to generate a plausible and good enough answer, this is send back as a response. The incoming query and this answer is also stored in another queue, to be picked up by the data pipeline later. This query based data is used to in/validate the database contents and estimate the position of previously unknown networks as to be near the already known networks.

I'll get familiar with the geolocate and fallback code, so I can make an informed technical recommendation on updating this text.